src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -52,15 +46,24 @@ export function getOptionsByDate(
   }, []);
 }
 
-function ErrorMessage({ facilityId, startRequestFlow }) {
+function startRequestFlow({ requestAppointmentDateChoice, history }) {
+  return requestAppointmentDateChoice(history);
+}
+
+function ErrorMessage({ facilityId, requestAppointmentDateChoice }) {
   return (
     <div aria-atomic="true" aria-live="assertive">
       <AlertBox
         status="error"
         headline="We’ve run into a problem when trying to find available appointment times"
       >
         To schedule this appointment, you can{' '}
-        <button onClick={startRequestFlow} className="va-button-link">
+        <button
+          onClick={() =>
+            startRequestFlow({ requestAppointmentDateChoice, history })
+          }
```

Comment:
You should be able to replace this line with:

`onClick={() => requestAppointmentDateChoice(history)}`